std_msgs/String task
---
zeabus_vision/vision_casino_gate data